.circleci/config.yml

```diff
@@ -4,9 +4,57 @@ version: 2.1
 orbs:
   architect: giantswarm/architect@2.7.0
 
+
+jobs:
+  test-policies:
+    machine: true
+    environment:
+      KIND_VERSION: v0.10.0
+      KUBERNETES_VERSION: v1.19.7
+    steps:
+      - checkout
+      - run:
+          name: Download kind
+          command: |
+            curl -sLo kind https://github.com/kubernetes-sigs/kind/releases/download/$KIND_VERSION/kind-linux-amd64
+      - run:
+          name: Download kubectl
+          command: |
+            curl -sLO https://storage.googleapis.com/kubernetes-release/release/$KUBERNETES_VERSION/bin/linux/amd64/kubectl && chmod +x kind kubectl
+      - run:
+          name: Create cluster
+          command: |
+            ./kind create cluster --image quay.io/giantswarm/kind-node:$KUBERNETES_VERSION --name kyverno-cluster  2> /dev/null
+      - run:
+          name: Wait for the cluster node to be ready
+          command: |
+            ./kubectl wait nodes/kyverno-cluster-control-plane --for=condition=ready --timeout=5m > /dev/null
+            ./kind get kubeconfig --name kyverno-cluster > ./kube.config
+      - run:
+          name: Install Kyverno
+          command: |
+            ./kubectl create -f https://raw.githubusercontent.com/kyverno/kyverno/main/definitions/release/install.yaml > /dev/null
```

Comment:
IMO this could be also done in the test by creating the app cr through the `kubectl` helper. [Example](https://github.com/giantswarm/loki-app/pull/21/files#diff-4c9860d793d258a73b4ad0eb6c7be75ada8aedbcec5634d3f9dc8b42e3c30c53R132)

Reply:
I will do this in a follow up. Would I put it in a function which I then call in each test? Or is there a way to trigger a function before all other tests run?

Reply:
You described a fixture. https://docs.pytest.org/en/stable/reference.html#pytest-fixture-api

Reply:
Okay - trying that out in a follow up. Merging here to have something at least :)

Reply:
Just remembered as I merged this:
The kyverno-app contains the CRDs for the policies. So we can't install the kyverno app in the test because the policies app will be failed by then.

Reply:
You can instruct abs to skip deploying the app by setting `app-tests-skip-app-deploy: true` in the config file
Then you must take care of installing everything you need for the test by yourself.